### ISSUE TITLE
Getting BAD REQUEST  too quick in Firefox

### ISSUE THREAD

**Issue by bwakkie** [pgadmin-org/pgadmin4#7130]
When I start working with pgadmin on Firefox i relatively quickly get a ...
`"BAD REQUESTBAD REQUEST" `
...in the message pane and I have to relogin losing my query.

In chromium the browser when this is happening it asks to re-initiate or re-create a new session, but at least I can continue working.

(A little bit like > https://dba.stackexchange.com/questions/330270/postgresql-server-timeout-issue-with-pgadmin-pgadmin-web-and-dbeaver )

I am working on the same machine as the server might this be why? On my laptop I can work for a longer time without any issues. 

Is there a config setting I might have missed?

firefox: 122.0
chromium: 121
pgadmin: 8.2
 

EDIT: made text clearer?

**Comment by yogeshmahajan-1903** [comment]
@bwakkie 
Can you please share screen recording and pgadmin logs for the same?

**Comment by bwakkie** [comment]
after restarting the pgadmin client and relogin
After waiting for like 10 minutes the below error messages start appearing into the pgadmin logfile.

```
2024-02-05 18:27:33,942: ERROR  pgadmin:        400 Bad Request: The CSRF tokens do not match.
Traceback (most recent call last):
  File "/data/venv/pgadmin/pgadmin311/lib/python3.11/site-packages/flask_wtf/csrf.py", line 261, in protect
    validate_csrf(self._get_csrf_token())
  File "/data/venv/pgadmin/pgadmin311/lib/python3.11/site-packages/flask_wtf/csrf.py", line 115, in validate_csrf
    raise ValidationError("The CSRF tokens do not match.")
wtforms.validators.ValidationError: The CSRF tokens do not match.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/data/venv/pgadmin/pgadmin311/lib/python3.11/site-packages/flask/app.py", line 1482, in full_dispatch_request
    rv = self.preprocess_request()
         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/venv/pgadmin/pgadmin311/lib/python3.11/site-packages/flask/app.py", line 1974, in preprocess_request
    rv = self.ensure_sync(before_func)()
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/data/venv/pgadmin/pgadmin311/lib/python3.11/site-packages/flask_wtf/csrf.py", line 229, in csrf_protect
    self.protect()
  File "/data/venv/pgadmin/pgadmin311/lib/python3.11/site-packages/flask_wtf/csrf.py", line 264, in protect
    self._error_response(e.args[0])
  File "/data/venv/pgadmin/pgadmin311/lib/python3.11/site-packages/flask_wtf/csrf.py", line 307, in _error_response
    raise CSRFError(reason)
flask_wtf.csrf.CSRFError: 400 Bad Request: The CSRF tokens do not match.
```

Screen record is a bit overkill as not much is happening but ....
![pgadminbug](https://github.com/pgadmin-org/pgadmin4/assets/1162977/4d25dc38-0191-4657-a93d-666c9f7da368)
 .... I attach a screenshot of the part in the client including the tail of the log on the left.

**Comment by pravesh-sharma** [comment]
Hi @bwakkie,

Can we connect over a call/meeting? My timezone is IST.

**Comment by bwakkie** [comment]
Yes this is possible @pravesh-sharma I am at UTC+1.

**Comment by pravesh-sharma** [comment]
Hi @bwakkie,

We are available Mon - Thu between 4-6 PM UTC +5.30

**Comment by adityatoshniwal** [comment]
No response from the author. Closing.
